index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/defaultStyle.css">
+	<link rel="stylesheet" href="./styles/generalStyle.css">
+	<link rel="stylesheet" href="./styles/style.css">
+	<title>DiegoPortfolio</title>
+</head>
```

Comment:
- [ ] Great job adding the fonts from the Figma design 🌵, but when you deploy your project, the fonts will change, please add in the `<head>` of your `.html` file the CDN link, 🖼️  I suggest you use [Google Fonts.](https://fonts.google.com/). Also, you can check the rendered fonts if you inspect your project in the browser, press the `computed` tab at the end of it you will find the rendered font. 💯 

![image](https://user-images.githubusercontent.com/91301423/198421045-9bace145-822b-478a-813c-1f1cab9d7713.png)